shell/platform/darwin/ios/framework/Source/FlutterTextInputDelegate.h

```diff
@@ -59,7 +59,8 @@ typedef NS_ENUM(NSInteger, FlutterFloatingCursorDragState) {
     insertTextPlaceholderWithSize:(CGSize)size
                        withClient:(int)client;
 - (void)flutterTextInputView:(FlutterTextInputView*)textInputView removeTextPlaceholder:(int)client;
-- (void)flutterTextInputViewDidResignFirstResponder:(FlutterTextInputView*)textInputView;
+- (void)flutterTextInputViewDidResignFirstResponder:(FlutterTextInputView*)textInputView
```

Comment:
Agree with @cyanglaz here, it should be `flutterTextInputView:didResignFirstResponderWithClient` or something similar, that would also match the other methods in this file.

Reply:
nit: maybe `didResignFirstResponderWithTextInputClient` since it's unclear what a client is.